apps/asap-server/test/integration/fetch-users.integration-test.ts

```diff
@@ -3,16 +3,16 @@ import Chance from 'chance';
 import { Role, UserResponse } from '@asap-hub/model';
 import { User } from '@asap-hub/squidex';
 import Users from '../../src/controllers/users';
-import { createUser } from '../helpers/users';
+import { createUser, createRandomOrcid } from '../helpers/users';
 
 const chance = new Chance();
 const users = new Users();
 
 describe('Users', () => {
   test('Should create and fetch a user', async () => {
-    const randomOrcid = chance.ssn();
     const randomEmail = chance.email();
     const randomName = chance.guid();
+    const randomOrcid = createRandomOrcid();
```

Comment:
do we want to add a test with an invalid Orcid rejected by squidex?

Reply:
Have written the test for invalid orcids but on reflection do we want a test for this? isn't this like testing squidex itself (which we said we would avoid doing in a previous ticket)?

Reply:
well since it's the only way we can test that we actually have a validation in place i would say - sure, why not, leave it